compiler/qsc_partial_eval/src/evaluation_context.rs

```diff
@@ -87,12 +88,15 @@ impl Scope {
             callable,
             args_runtime_properties,
             env: Env::default(),
+            last_expr: None,
             hybrid_exprs: FxHashMap::default(),
             hybrid_vars: FxHashMap::default(),
         }
     }
 
-    pub fn get_expr_value(&self, expr_id: ExprId) -> &Value {
+    // Maybe this can be removed in favor of a single value getter, which would also eliminate the need
+    // for `hybrid_exprs` entirely?
+    pub fn _get_expr_value(&self, expr_id: ExprId) -> &Value {
```

Comment:
We should talk through this idea... I'm more and more convinced that the expr-to-value hashmap shouldn't be needed.

Reply:
After some discussion, we landed at leaving the hashmap in right now to support ongoing work and reevaluating if ongoing work needs to use `get_expr_value` for more than the most recent expression result.